src/v3/test/integration/authenticator-enroll-email-magic-link-false.test.tsx

```diff
@@ -35,6 +36,7 @@ describe('Email authenticator enroll when magic link = false Tests', () => {
     });
 
     const headerEle = await findByRole('heading', { level: 2 });
+    await waitFor(() => expect(headerEle).toHaveFocus());
```

Comment:
Need to wait for Title to have focus first, or else it will take away focus from the input field, causing it to fail to type the mock input